modules/countdown.js

```diff
@@ -4,21 +4,52 @@ require('moment-countdown')
 let timeout = null
 
 const ARTICLE_50 = '2020-01-31T00:00:00+01:00'
+const TRANSITION = '2020-12-31T00:00:00+01:00'
 
 function autoCount (bot, lastTick) {
-  const thisTick = moment(ARTICLE_50).countdown().toString().split(/, | and /)[0]
+  const now = moment()
+
+  var eventName
```

Comment:
You could initialise these variables to the transition period values, and then overwrite them if it's still Article 50.